Rust/src/trcf/predictorcorrector.rs

```diff
@@ -149,7 +149,7 @@ impl PredictorCorrector {
                 }
                 let internal_timestamp = result.internal_timestamp;
                 let base_dimension = forest.dimensions() / shingle_size;
-                let mut start_position = (shingle_size - 1) * base_dimension;
+                let start_position = (shingle_size - 1) * base_dimension;
```

Comment:
This whole line should removed -- it seems start_position has been redefined in all of the branches. But removing the mut makes sense.